opentelemetry-otlp/Cargo.toml

```diff
@@ -16,17 +16,31 @@ license = "Apache-2.0"
 edition = "2018"
 build = "build.rs"
 
+[lib]
+doctest = false
```

Comment:
why skip doctest?

Reply:
[`prost`](https://crates.io/crates/prost) retains comments from .proto files in generated Rust code. Comments in generated Rust code from .proto files breaks Rust `doctest`. 

Retaining comments from .proto files is an unusual and unique feature of [`prost`](https://crates.io/crates/prost). When generating Java or Go Golang code from .proto files, comments are usually not being retained.

https://github.com/danburkert/prost/pull/389 introduces a new option to disable comment retention from .proto files in generated Rust code. Awaiting upstream change. I'm keeping an eye on that.